exec/java-exec/src/test/java/org/apache/drill/exec/work/metadata/TestMetadataProvider.java

```diff
@@ -211,12 +217,15 @@ public void tablesWithTableNameFilter() throws Exception {
 
     assertEquals(RequestStatus.OK, resp.getStatus());
     List<TableMetadata> tables = resp.getTablesList();
-    assertEquals(5, tables.size());
+    assertEquals(7, tables.size());
 
     verifyTable("sys", "boot", tables);
     verifyTable("sys", "memory", tables);
     verifyTable("sys", "options", tables);
+    verifyTable("sys", "options2", tables);
     verifyTable("sys", "version", tables);
+    verifyTable("sys", "internal_options", tables);
+    verifyTable("sys", "internal_options2", tables);
```

Comment:
Better name? Nobody is going to know what "options2" means... I certainly can't guess...